bin/ci-import

```diff
@@ -28,7 +28,7 @@ waitingPid=$!
 wait "$waitingPid"
 waitingReturnCode=$?
 # clean up PHP processes
-kill "$workerPid" "$queueWatchPid";
+kill "$workerPid" "$queueWatchPid" || true;
 if [ ! "$waitingReturnCode" -eq "0" ]; then
     echo "Waiting for queues to be empty timed out"
```

Comment:
and also substitute the echo with `echo "./wait-for-empty-gearman-queue timed out" to be more clear`